client/context/Base.tsx

```diff
@@ -12,47 +12,107 @@ export type Club = {
   id: Id
   name: string
   description: string
+  backgroundImageURL?: string
+  iconURL: string
+  facebookLink?: string
+  twitterLink?: string
+  instagramLink?: string
+  websiteLink?: string
+  tags: Array<string>
+  events: Array<BasicEvent>
+}
+
+export interface BasicEvent {
+  name: string
+  startTime: string
+  endTime: string
+  backgroundImageURL?: string
+  tags: Array<string>
 }
 
 export type Event = {
   id: Id
-  name: string
-  description: string
+  name: string,
+  club: string,
```

Comment:
should be a `BasicClub` based on `https://github.com/loolabs/waterpark/pull/71/files#diff-469d1105c94fbfbe41d10aa21f68b1de04725545cf788b8ca5b18ea6afbd0395R3-R16`